packages/webapp/src/components/CardsCarrousel/styles.module.scss

```diff
@@ -39,22 +39,49 @@
   }
 }
 
+@keyframes card-to-front {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+@keyframes card-swapping {
+  from {
+    opacity: 0.5;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
 .activeCard {
   border-radius: 4px;
   flex-grow: 1;
+  z-index: 999;
+  animation: card-swapping 300ms ease-out;
 }
 
 .inactiveCard {
   border-radius: 0 4px 4px 0;
   padding-right: 8px;
-  margin-left: -2px;
+  animation: card-swapping 300ms ease-out;
 
   @media only screen and (min-width: 1024px) {
     padding-right: 20px;
     padding-left: 24px;
   }
 }
 
+.inactiveCardButton {
+  height: 100%;
+  width: calc(100% - 2px);
+  overflow: hidden;
```

Comment:
For me this cuts off the icons on the left. Is that the intended behaviour?

![Screenshot 2023-11-06 at 11 17 11 AM](https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/e326ffc9-ba1c-4bc9-a11c-c36f89de7179)

Reply:
It was intended to cut them off in mobile when the cards are more overlapped, but they shouldn't look like this in desktop! Will take a look